app/views/bets/_display_bets.html.erb

```diff
@@ -5,10 +5,22 @@
         <tr>
           <th class="col-md-2">Created By</th>
           <th class="col-md-2">Created at</th>
+          <th class="col-md-2">Expires at</th>
           <th class="col-md-2">Total amount bet</th>
-          <th class="col-md-6">Bet description</th>
+          <th class="col-md-4">Bet description</th>
         </tr>
       </thead>
+      <tbody>
+        <% bets.each do |bet| %>
```

Comment:
Thanks for the feedback! Yes, it's the one that displays the bets on the screen, depending on how many have been passed to \@bets.